docs/faqs.md

```diff
@@ -372,6 +373,13 @@ Conan will build from sources all the packages and use the shared libraries when
 > [`shared_library_package_id`](https://docs.conan.io/en/latest/reference/conanfile/methods.html?highlight=shared_library_package_id#self-info-shared-library-package-id),
 > that will encode this information in the package ID and ensure that any change in the static libraries that are
 > embedded into a shared one is taken into account when computing the package ID.
-> 
+>
 > In this repository we are not using it, because it will lead to many missing packages, making it impossible
 > for the CI to actually build consumers in PRs.
+
+## How to watch only specific recipes?
+
+By default, Github allows us to receive notifications for any new PR, or PRs where we are mentioned directly, or even when added as a reviewer.
+However, if you want to subscribe to review or even watch one or more recipes and avoid receiving emails for any new pull request in Conan Center Index,
+you should add a new entry in the file `.github/CODEOWNERS`, which is a [Github feature to customize notifications](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
+Remember that it's just a feature name, the Conan Center Index has no maintainers, nor owners for recipes, all of them are maintained by the whole community.
```

Comment:
Please add here content following next lines:
 * We cannot enable write-access to contributors, sorry, feature won't work out-of-the-box, but still we think this is the right place to store this information
 * The content of this file can be parsed by some future gh-action to notify subscribers
 *